polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -245,15 +247,15 @@ class PythonInterpreter(val symbolTable: RuntimeSymbolTable) extends LanguageKer
         val numElements = jep.getValue(s"len($accessor)", classOf[java.lang.Number]).longValue()
         val (elems, types) = (0L until numElements).map(n => getPyResult(s"$accessor[$n]")).toList.unzip
 
-        val listType = symbolTable.global.ask {
+        val listType = global.ask {
           () =>
             val lubType = types.flatten.toList match {
               case Nil      => typeOf[Any]
-              case typeList => try symbolTable.global.lub(typeList) catch {
+              case typeList => try kernelContext.global.lub(typeList) catch {
                 case err: Throwable => typeOf[Any]
               }
             }
-            symbolTable.global.appliedType(typeOf[List[Any]].typeConstructor, lubType)
+            kernelContext.global.appliedType(typeOf[List[Any]].typeConstructor, lubType)
```

Comment:
Should we be consistent with our references to `global`? In some places we do `kernelContext.global` and in others we just use `global`.

Reply:
Yes, absolutely. Fixed.